vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -345,11 +350,11 @@ private String generatePodSpec(Map config) {
                                     emptyDir: [:]
                                 ]]
     }
-
+    def test = new JsonUtils().groovyObjectToPrettyJsonString(podSpec)
+    echo "podSpec : ${test}"
```

Comment:
Is this echo needed or it has been added during debugging this code?

Reply:
echo wasn't needed. It is removed